md4c/md4c.h

```diff
@@ -129,7 +129,13 @@ typedef enum MD_SPANTYPE {
     /* <del>...</del>
      * Note: Recognized only when MD_FLAG_STRIKETHROUGH is enabled.
      */
-    MD_SPAN_DEL
+    MD_SPAN_DEL,
+
+    /* For recognizing inline ($) and display ($$) equations
+     * Note: Recognized only when MD_FLAG_LATEX is enabled.
+     */
+    MD_SPAN_LATEX,
+    MD_SPAN_LATEX_DISPLAY
```

Comment:
Please rename to `MD_SPAN_LATEXMATH` and `MD_SPAN_LATEXMATH_DISPLAY`. As far as I can understand it, this does not allow embedding any LaTeX snippets, only math ones, right?

Reply:
Yes. This will only pick up on $, $$ delimiters so only on equations. So makes sense.